spectrum_io/raw/msraw.py

```diff
@@ -77,7 +77,7 @@ def read_mzml(
                     fragmentation = spec["scanList"]["scan"][0]["filter string"].split("@")[1][:3]
                     mz_range = spec["scanList"]["scan"][0]["filter string"].split("[")[1][:-1]
                     key = f"{file_name}_{id}"
-                    if search_type == "Maxquant":
+                    if search_type.lower() == "maxquant":
```

Comment:
no, please only do this once at the beginning of the function, i.e. "search_type = search_type.lower()".
Also, please change the default to "maxquant" instead of "Maxquant"